mesa/time.py

```diff
@@ -60,12 +60,19 @@ class BaseScheduler:
         - agents (property): Returns a list of all agent instances.
     """
 
-    def __init__(self, model: Model) -> None:
+    def __init__(self, model: Model, agents: Iterable[Agent] | None = None) -> None:
```

Comment:
Does this also mean you can now initialize an scheduler with an existing `AgentSet`? If so, great feature, please update the docs that you can now do this.